calculator/operations.py

```diff
@@ -1,23 +1,19 @@
+import functools
+
 
 def add(*args):
-    # your implementation here
-    pass
+    return sum(args)
 
 def subtract(*args):
-    # your implementation here
-    pass
+    return functools.reduce(lambda x, y: x-y, args)
```

Comment:
Your lambda functions are so simple and beautiful. Gotta get more comfortable with those myself!